spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -71,10 +78,16 @@ definitions:
 
  - MSG_POS_ECEF:
     id: 0x0200
-    short_desc: Position in ECEF
+    public: True
+    short_desc: Single-point position in ECEF
     desc: |
-      Position solution in absolute Earth Centered Earth Fixed (ECEF)
-      coordinates.
+      The single-point position solution message reports absolute
+      Earth Centered Earth Fixed (ECEF) coordinates and the status
+      (single point absolute vs RTK) of the position solution. If the
+      rover receiver knows the single-point point position of the base
+      station and has an RTK solution, this reports a pseudo-absolute
```

Comment:
This should say "Surveyed position of the base station".  The user has to know an accurate position of the base station in order for the absolute RTK positions to be correspondingly accurate.